viz_scripts/docker/load_mongodump.sh

```diff
@@ -1,9 +1,71 @@
+#!/bin/bash
+
+# Directory of the script
+SCRIPT_DIR="$(dirname "$0")"
+
+# Path to the configuration file (one level up)
+CONFIG_FILE="$SCRIPT_DIR/../../docker-compose.dev.yml"
+
+# Check if the correct number of arguments is provided
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <mongodump-file>"
+    echo "  <mongodump-file> : The path to the MongoDB dump file to be restored."
+    echo " run git add -f <docker compose file> after using this command"
```

Comment:
Please remove this since it contradicts the README instructions. I know it was in the original PR, but was removed in response to review feedback.

Reply:
@iantei the entire if check does not contradict the README, only the last `echo`. I will fix this before merging so that we don't have to wait for yet another round of reviews.

Reply:
I have re-introduced the check while removing the last `echo` statement.